src/data/markdown/docs/40 xk6-disruptor/01 Get started/01 Welcome.md

```diff
@@ -5,46 +5,76 @@ head_title: 'xk6-disruptor Documentation'
-These characteristics make it easier to incorporate such tests into the test suites of applications that deploy on shared infrastructures (such as staging environments).
+Other types of faults and disruptors will be introduced in the future. The [Roadmap](https://github.com/grafana/xk6-disruptor/blob/main/ROADMAP.md) presents the project's goals for the coming months regarding new functionalities and enhancements.
 
+```javascript
+export default function () {
+    // Create a new pod disruptor with a selector
+    // that matches pods from the "default" namespace with the label "app=my-app"
+    const disruptor = new PodDisruptor({
+        namespace: "default",
+        select: { labels: { app: "my-app" } },
+    });
 
-<Blockquote mod="attention">
+    // Check that there is at least one target
+    const targets = disruptor.targets();
+    if (targets.length === 0) {
+        throw new Error("expected list to have one target");
+    }
 
-xk6-disruptor is in the alpha phase, undergoing active development. k6 doesn't guarantee API compatibility between releases.
-Until this extension reaches v1.0 release, your k6 scripts may need to be updated on each release .
+    // Disrupt the targets by injecting HTTP faults into them for 30 seconds
+    const fault = {
+        averageDelay: 500,
+        errorRate: 0.1,
+        errorCode: 500
+    }
+    disruptor.injectHTTPFaults(fault, "30s")
+}
+```
 
-</Blockquote>
+## Use cases
 
-## Try out xk6-disruptor on our demo environment
+The disruptor lets you test the resiliency of distributed applications by introducing errors in the requests served by your services.
 
-We provide an [interactive demo environment in Killercoda](https://killercoda.com/grafana-xk6-disruptor/scenario/killercoda) which you can use to try xk6-disruptor right away without having to install Kubernetes on your local machine.
+The disruptor does not try to reproduce root causes, such as failed application instances or degraded computing or network resources. 
+It focuses on reproducing the side effects of such failures, so you can focus on understanding the propagation of errors between internal and public services and improving the error handling in your application. 
 
-You can use this sandbox to try out xk6-disruptor and start writing fault injection on a microservices application right away.
+This way, the disruptor makes reliability tests repeatable and predictable while limiting their blast radius. 
+These are essentials to test applications deployed on shared infrastructures such as pre-production and testing environments.
 
-## Install xk6-disruptor on your environment
+Common use cases are:
+- Test resilient policies such as backoff, timeouts, retries, etc.
+- Test the fallback functionality when internal failures arise.
+- Test SLOs under common internal failures.
+- Test application performance when experiencing delays.
+- Add fault injection to existing performance tests.
```

Comment:
Should this be in bare infinitive? Gerund or infinitive with "to" sounds slightly better to me.

Reply:
[Grafana Writer toolkit](https://grafana.com/docs/writers-toolkit/style-guide/) recommend direct, concise, and active voice. 

The infinitive seems appropriate here.

Reply:
Need to merge this PR. @heitortsergent, please, consider it for later changes.